app/components-react/widgets/ChatBox.tsx

```diff
@@ -147,7 +136,7 @@ export class ChatBoxModule extends WidgetModule<IChatBoxState> {
     };
 
     if (this.config.useNewWidgetAPI) {
-      return { global: obj };
+      return settingsToGlobal(obj);
```

Comment:
i think you could avoid this additional check by doing `super.patchBeforeSend(obj)` here but thats pretty nitpicky